service/features/Steps/customers_steps.py

```diff
@@ -0,0 +1,33 @@
+"""
```

Comment:
I'm not quite sure if this is intentional but you've copy and pasted the `/features` directory into the `/service` directory??

There should not be a `/service/features` directory at all. You can delete the whole things